doc/installation_instructions/platform.rst

```diff
@@ -1,30 +1,20 @@
-.. _install-notes:
+.. _platform-installation:
 
-Installation on specific platforms
-==================================
+Platform Install Guide
+======================
 
-The following describes installation details for various systems and platforms
-that SmartSim may be used on.
-
-.. include:: platform/generic.rst
-
-.. include:: platform/nonroot-linux.rst
+HPC platforms have specific modules that users can often use so they do not
+need to retrieve all of the build dependencies themselves. Some machines
```

Comment:
`Some machines have specific environment variables and/or configuration settings that need to be set for optimal performance.` -> `Some machines require specific environment variables and/or configuration settings for optimal performance`